crates/bevy_text/src/text2d.rs

```diff
@@ -72,6 +72,7 @@ pub struct Text2dBundle {
     pub visibility: Visibility,
     /// Algorithmically-computed indication of whether an entity is visible and should be extracted for rendering.
     pub computed_visibility: ComputedVisibility,
+    pub text_layout_info: TextLayoutInfo,
```

Comment:
could you add doc to this fields? the rest of this struct is already fully documented